app/main.py

```diff
@@ -1 +1,28 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import (
+    NotVaccinatedError,
+    OutdatedVaccineError,
+    NotWearingMaskError
+)
+
+
+def go_to_cafe(cafe: Cafe, friends: list) -> str:
```

Comment:
Indicate that you expect to obtain a list of dict for `friends` in the type annotation